### DISCUSSION TITLE
Why does 'Aborted' occur

### DISCUSSION THREAD

**Discussion by Zhoujieliang** [NCAR/noahmp#106] [Q&A]
After entering './hrldas.exe namelist.hrldas', the model runs normally.
![terme](https://github.com/NCAR/noahmp/assets/125653764/d710b2fb-82b4-4f8e-bfa0-a8b58001d5f1)
But after running for 20 days, an error occurred.
![terme2](https://github.com/NCAR/noahmp/assets/125653764/056b64de-bba8-4f7f-a0dc-cda01782483a)
This situation occurred twice yesterday, both with the same error message.
Do you know what the problem is. 
Thanks

**Comment by cenlinhe** [comment]
Please try reducing your model timestep from 1 hour to 30min.

**Reply by Zhoujieliang** [reply]
My original model timestep is 3600 seconds, but it stopped running with an error message on January 20th. 
After listening to your suggestion, I changed the model timestep to 1800 seconds, and it stopped moving until January 22nd.
![image](https://github.com/NCAR/noahmp/assets/125653764/903a560a-df00-4b5b-9ede-cf4865e090da)

**Comment by cenlinhe** [comment]
Could you please try further reducing the model timestep to 900 seconds? If this still does not solve the problem, this may indicate that you have one or more points with bad atmospheric forcing values. What forcing data are you using? Could you please also provide your namelist?

**Reply by Zhoujieliang** [reply]
I set the timestep to 900 seconds,but the problem still not solve. It aborted in 2015-01-23_11:00:00.
The forcing data is GLDAS.
namelist.hrldas like that.
![1](https://github.com/NCAR/noahmp/assets/125653764/21ce6b5a-f212-48ab-aa91-c2c0c5831196)
![2](https://github.com/NCAR/noahmp/assets/125653764/1900f760-265c-4f26-a6f2-2f9a0fdf9636)

Also, namelist.input.GLDAS like that.
![name GLDAS](https://github.com/NCAR/noahmp/assets/125653764/b267dcd4-abec-42f5-bf89-742cdcaa6d86)

**Comment by cenlinhe** [comment]
you may want to check your forcing files for 2015-01-20 ~ 2015-01-23 for each forcing variables to see if there is any weird or NaN values in grids where it shouldn't occur. What is your simulation domain (e.g., does it include a lot of coastal grids or any small island grids)? Another test could be increasing the EXPAND_LOOP from 5 to 10 to re-generate your forcing files for that month and re-run the simulation to see if it solves the problem.
